lightning/src/ln/msgs.rs

```diff
@@ -570,6 +570,77 @@ pub struct ChannelUpdate {
 	pub contents: UnsignedChannelUpdate,
 }
 
+/// A query_channel_range message that can be sent or received from a peer
```

Comment:
You can explain shortly purpose of this message, what response you aim to provoke sending it to your peer to fulfill which need.

Reply:
I updated comments for remove the old standard message and added general description of what the message does, what the reply should be, and any notes on serialization limitations.  Please let me know if it's too verbose now or if there is anything that is unclear, thanks!